### PR TITLE
Dominic Torruellas - JavaScript II

### PR DESCRIPTION
**Pull request by djtorel**

### CODE REVIEW

**Comment by djtorel** [comment]
MVP Complete, for real this time

File: assignments/array-methods.js [hunk 1]
```diff
@@ -1,83 +1,517 @@
...
+  []
+);
+console.log(nameAndCompany);
 
-// Problem 3
\ No newline at end of file
+// Problem 3 - Return a list of people who  donated more than 250
+const highRollers = runners.filter(({ donation }) => donation >= 250);
+console.log(highRollers);
+// mvp complete
```
Comment (Dilstom) [inline assignments/array-methods.js:517]: perfect!

File: assignments/callbacks.js [hunk 2]
```diff
@@ -17,39 +20,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
...
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+  const newArray = array.reduce(
+    (newArr, value) => (newArr.includes(value) ? newArr : [...newArr, value]),
+    []
+  );
+  return cb(newArray);
 }
+removeDuplicates([1, 2, 1, 3, 4, 3], result => console.log(result));
```
Comment (Dilstom) [inline assignments/callbacks.js:77]: great job with callbacks!

File: assignments/closure.js [hunk 3]
```diff
@@ -1,21 +1,44 @@
...
 };
+
+console.log('Factory');
+const newCounterFactory = counterFactory();
+console.log(newCounterFactory.increment());
+console.log(newCounterFactory.increment());
+console.log(newCounterFactory.increment());
+console.log(newCounterFactory.decrement());
+console.log(newCounterFactory.decrement());
+console.log(newCounterFactory.decrement());
```
Comment (Dilstom) [inline assignments/closure.js:44]: Awesome job, Dominic!

**Review by Dilstom — APPROVED** [review]

**Pull request merged**